modules/govuk/manifests/node/s_api_lb.pp

```diff
@@ -60,6 +61,14 @@
     internal_only => true,
   }
 
+  if !empty($mapit_servers) {
+    loadbalancer::balance { 'mapit':
+      servers       => $mapit_servers,
+      internal_only => true,
+      https_only    => false, # FIXME: Remove for #51136581
```

Comment:
Makes sense to me!

I think that ID is a story in the backlog that infrastructure had a long long time ago, so I'll go through separately and tidy that up :)